src/dao/AppDAO.js

```diff
@@ -91,7 +93,7 @@ class AppDAO extends AbstractContractDAO {
     };
 
     getLOCbyID = (index: number, account: string) => {
-        return this.contract.then(deployed => deployed.getLOCbyID.call({index, from: account}));
+        return this.contract.then(deployed => deployed.getLOCbyID.call(index, {from: account}));
     };
```

Comment:
- move LOC logic out of AppDAO
- you should return LOCModel from this function, not just a result of web3 function